packages/contracts/contracts/plugin/PluginRepo.sol

```diff
@@ -196,11 +176,44 @@ contract PluginRepo is
         Tag memory tag = Tag(_release, build);
         bytes32 _tagHash = tagHash(tag);
 
-        versions[_tagHash] = Version(tag, _pluginSetup, _contentURI);
+        versions[_tagHash] = Version(tag, _pluginSetup, _buildMetadata);
 
         latestTagHashForPluginSetup[_pluginSetup] = _tagHash;
 
-        emit VersionCreated(_release, build, _pluginSetup, _contentURI);
+        emit VersionCreated(_release, build, _pluginSetup, _buildMetadata);
+
+        if (_releaseMetadata.length > 0) {
```

Comment:
Shouldn't we also have a check that reverts if this is the first build of a release, and there's no release metadata?

Reply:
I revert it.  check this line in createVersion.

```
if (_releaseMetadata.length == 0) {
                revert ReleaseMetadataInvalid({release: _release, metadata: _releaseMetadata});
            }
            ```

Reply:
Sorry, my bad :+1: